nautobot/ipam/filters.py

```diff
@@ -222,53 +222,31 @@ class Meta:
         fields = ["date_allocated", "id", "prefix_length", "tags"]
 
     def filter_prefix(self, queryset, name, value):
-        value = value.strip()
-        if not value:
-            return queryset
-        # filter for Prefix models equal to |value|
-        try:
-            return queryset.net_equals(netaddr.IPNetwork(value))
-        except (AddrFormatError, ValueError):
-            return queryset.none()
+        prefixes = [value.strip() for value in value]
+        return queryset.net_equals(*prefixes)
 
     def search_within(self, queryset, name, value):
-        value = value.strip()
-        if not value:
-            return queryset
-        try:
-            return queryset.net_contained(netaddr.IPNetwork(value))
-        except (AddrFormatError, ValueError):
-            return queryset.none()
+        prefixes = [value.strip() for value in value]
```

Comment:
Here and below, do we need to throw away values that strip to an empty string as we were doing in the old code? Or is that being handled somewhere else?